### ISSUE TITLE
Keyboard navigation fails on opening screen (Chromebook)

### ISSUE THREAD

**Issue by proxysplsplsplssc** [nautilus-os/NautilusOS#85]
im on a Chromebook, and on the one file download it doesnt work when i use my arrow keys to change my selection and the enter doesnt work to chose my selection. please fix i really wanna use this
